models/IdentifiableEntity_model.py

```diff
@@ -0,0 +1,9 @@
+from controllers import db
```

Comment:
Python style is to have all file names in snake case, file should be identifiable_entity_model.py

Reply:
amended